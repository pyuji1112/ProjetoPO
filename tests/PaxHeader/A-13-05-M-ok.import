30 mtime=1635415031.905880134
