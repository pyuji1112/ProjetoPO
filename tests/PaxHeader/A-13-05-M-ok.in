30 mtime=1635415054.657839084
