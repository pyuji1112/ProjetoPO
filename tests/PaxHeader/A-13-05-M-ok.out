30 mtime=1635415054.227219523
